src/DynamoCoreWpf/ViewModels/Core/WorkspaceViewModel.cs

```diff
@@ -547,25 +547,6 @@ internal void Save(string filePath, bool isBackup = false, EngineController engi
             try
             {
                 // Stage 1: Serialize the workspace.
-                string fileName = string.Empty;
```

Comment:
I am not sure whether this should be done in Model. `Model.Save` just saves the compute and not the view part.  the other question is whether a dyn with compute part is useful for a client {All the view properties are initialized to defaults}. Did we test that use case?  If DynamoCLI can create a custom node (I guess it should) and we have use cases that we need to support, then doing this in model makes sense.

Reply:
@ramramps the name is part of the model, not the view.
```{
  "Uuid": "8de26b82-e2ab-442e-aa09-9a9c662353dd",
  "IsCustomNode": false,
  "Description": "",
  "Name": "Home",
  "ElementResolver": {
    "ResolutionMap": {}
  }, 
```
There are clients that might not need a view at all and all properties should be correct for them.
CLI can create custom nodes or do anything else Dynamo model can do pretty much.

Reply:
thanks @mjkkirschner